cmd/cli/kubectl-kyverno/test/manifest/print.go

```diff
@@ -0,0 +1,7 @@
+package manifest
+
+import "fmt"
+
+func print(manifest string) {
+	fmt.Println(manifest)
+}
```

Comment:
Not sure if I understand the point of this small function. Why not call fmt.Println directly?

Reply:
I’m thinking about using real template in the future, right now it’s just static content :(